Assets/_Application/Layers/LayerTypes/CartesianTiles/CartesianTileLayerGameObject.cs

```diff
@@ -49,6 +65,26 @@ private void SetupFeatures()
         {
             if (layer is not BinaryMeshLayer binaryMeshLayer) return;
 
+            //we have to apply styling when mappings are created, before we cannot load the values like from awake
+            binaryMeshLayer.OnMappingCreated.AddListener(m => ApplyStyling());
+            binaryMeshLayer.OnMappingCreated.AddListener(mapping =>
```

Comment:
Waarom Lambda's en niet concrete methods zodat je in de OnDestroy specifiek deze weg kan halen ipv een RemoveAllListeners?